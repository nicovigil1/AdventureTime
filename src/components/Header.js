import React from 'react'

export default class Header extends React.Component {
  render () {
    return(
      <div style={navStyle}>
        Welcome to AdventureTime
      </div>
    )
  }
}

const navStyle = {
  display: "flex",
  backgroundColor: "#63b4f4",
  justifyContent: "center",
  color: "black",
  paddingTop: ".5em",
  paddingBottom: ".5em",
  boxShadow: "0px 0px 10px 5px #000"
}