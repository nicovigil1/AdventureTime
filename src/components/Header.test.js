import React from 'react';
import Header from './Header';
import {shallow} from 'enzyme';
import { isMainThread } from 'worker_threads';

describe('Header', () => {
  let wrapper

  beforeEach(() => {
    wrapper = shallow(<Header />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})