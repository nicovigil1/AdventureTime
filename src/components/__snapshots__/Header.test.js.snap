// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should match the snapshot 1`] = `
<div
  style={
    Object {
      "backgroundColor": "#63b4f4",
      "boxShadow": "0px 0px 10px 5px #000",
      "color": "black",
      "display": "flex",
      "justifyContent": "center",
      "paddingBottom": ".5em",
      "paddingTop": ".5em",
    }
  }
>
  Welcome to AdventureTime
</div>
`;
